aptos-move/framework/aptos-framework/sources/fungible_asset.move

```diff
@@ -1311,13 +1321,13 @@ module aptos_framework::fungible_asset {
         mutate_metadata(
```

Comment:
Can you add a couple of test cases for trying to exceed the different limits?

Reply:
Added 🫡